keanu-python/keanu/algorithm/sampling.py

```diff
@@ -61,6 +71,13 @@ def __init__(self,
 
 
 class NUTSSampler(PosteriorSamplingAlgorithm):
+    """
+    :param adapt_count: The number of samples for which the step size will be tuned. For the remaining samples in which it is not tuned, the step size will be frozen to its last calculated value.
+    :param target_acceptance_prob: The target acceptance probability. A suggested value of this is 0.65 (Beskos et al., 2010; Neal, 2011).
```

Comment:
Shall we call out what the acceptance probability defaults to if this is left as none?